### PR TITLE
Simplified Chinese trsls

### PR DESCRIPTION
**Pull request by ChuijkYahus**
Co-Authored-by: @Dr-WeiAL

### CODE REVIEW

**Comment by ChuijkYahus** [comment]
Supports following mods' planks & leaves:

Mod Name|Footnote
-|-
The Aether|Done by @Dr-WeiAL
AoA3
Apple Trees Revived
Architect's Palette
Ars Elemental
Ars Nouveau
Biome Makeover
BOP|Done by @Dr-WeiAL
Blue Skies|Done by @Dr-WeiAL
Botania|Done by @Dr-WeiAL
BYG|Done by @Dr-WeiAL
Caupona
Cinderscapes
Croptopia|Done by @Dr-WeiAL
Cultural Delights
Darker Depths
Domum Ornamentum
Ecologics
Enhanced Mushrooms
Eternal Tales
Extended Mushrooms
F&A
ForceCraft
Fruit Trees
Greek Fantasy
Hex Casting
Hexerei
ID
Malum
Kitteh6660's MoreCraft
Mystical World
Nether's Exoticism
Enlightened End
Occultism
End's Phantasm
PizzaCraft
Premium Wood
Primal Magick
Project Brazier
Pyromancer
Quark|Done by @Dr-WeiAL
Project Rankine
Sakura
TiC
Terra Incognita
Terraqueous
TofuCraftReloaded
Twigs
Twilight Forest|Done by @Dr-WeiAL
Undergarden
Wilds
Windswept

basically everything i can find on cn wiki site for mods

**Comment by MehVahdJukaar** [comment]
Thanks a lot!
